tf-psa-crypto/drivers/builtin/include/mbedtls/error_common.h

```diff
@@ -1,14 +1,14 @@
 /**
- * \file error.h
+ * \file error_common.h
  *
- * \brief Error to string translation
+ * \brief Error codes
  */
 /*
  *  Copyright The Mbed TLS Contributors
  *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
  */
-#ifndef MBEDTLS_ERROR_H
-#define MBEDTLS_ERROR_H
+#ifndef MBEDTLS_ERRORCOMMON_H
```

Comment:
With the last version of generated_errors.pl we should be able to just have MBEDTLS_ERROR_COMMON_H here, right?